README.rst

```diff
@@ -134,16 +134,23 @@ Getting started
 
    .. code:: python
 
-    AUTHENTICATION_BACKENDS += ["sesame.backends.ModelBackend"]
+    AUTHENTICATION_BACKENDS = [
+        ...,
+        "sesame.backends.ModelBackend",
+        ...,
```

Comment:
If there's no ordering constraint with django's built-in `ModelBackend` then I think it's best to just leave this as ellipsises to avoid any confusion around whether `ModelBackend` is strictly required.

Reply:
There were three reasons behind the ready-to-use example, including both backends, that I suggested:

1. Most projects don't override `AUTHENTICATION_BACKENDS`. It isn't in the default generated settings file (unlike `MIDDLEWARE`). By giving the full content, I save users a lookup of the default value in Django's docs.
2. This discussion started from — it is too easy to replace `"django.contrib.auth.backends.ModelBackend"` with `"sesame.backends.ModelBackend"` instead of adding `"sesame.backends.ModelBackend"`. We wanted to clarify that the most common usage is to include both.
3. While it is technically possible to remove `"django.contrib.auth.backends.ModelBackend"` from `AUTHENTICATION_BACKENDS` and`"django.contrib.auth.middleware.AuthenticationMiddleware"` from `MIDDLEWARE`, this isn't very common. Often you'll keep them at least for the admin.

----

If we wanted to provide additional guidance in prose, it would be:

> Since `"django.contrib.auth.backends.ModelBackend"` and `"sesame.backends.ModelBackend"` take different parameters, no set of credentials is valid for both, so the order doesn't matter.

I feel like this would create more confusion than clarity and those who want the answer can find it easily in Django's docs, which state:

> The order of [AUTHENTICATION_BACKENDS](https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-AUTHENTICATION_BACKENDS) matters, so if the same username and password is valid in multiple backends, Django will stop processing at the first positive match.